kafka-workitem/pom.xml

```diff
@@ -0,0 +1,173 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jbpm.contrib</groupId>
+    <artifactId>workitems</artifactId>
+    <version>7.14.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>kafka-workitem</artifactId>
+  <name>Kafka</name>
+  <description>publish messages from a process</description>
+   <organization>
+    <name>jBPM dev team</name>
```

Comment:
Change this to include your name, email or your organization name (or however you want to be presented here as contributer)

Reply:
Thanks for creating a JIRA ticket.